### DISCUSSION TITLE
Detection Playbooks - Alerts

### DISCUSSION THREAD

**Discussion by RoyTM99** [Security-Onion-Solutions/securityonion#4420] [Unsupported Versions] [answered]
Hi Everyone,

I am implementing the Security Onion solution in an environment with about 40 servers/network devices/antivirus, I am trying to generate some alerts related to playbooks, I have already enabled a few but I do not see any related alerts activated, is it possible that something additional is missing or for example where it is advisable to activate alerts such as "port scan" "malware detection"?

The data sources are already connected and we are receiving logs which we can see through kibana.

Elast Version: 7.10.2
Security Onion: 2.3.30

Thank you
Kind Regards

**Comment by EchoGangster** [comment, marked as answer]
Hello there.  There are additional rules you can enable on the SO-Manager then activate them in Playbooks. 
https://docs.securityonion.net/en/2.3/playbook.html?highlight=compliance#adding-additional-rulesets

Adding Additional Rulesets

As previously mentioned, the pre-loaded Plays come from the community Sigma repository (https://github.com/Neo23x0/sigma/tree/master/rules). The default config is to only pull in the Windows rules. The rest of the rules from the community repository can be pulled in by editing a pillar value under /opt/so/saltstack/local/pillar/global.sls

soctopus:
playbook:
rulesets:
windows
Add one or more of the following:

application,apt,cloud,compliance,generic,linux,network,proxy,web

These are based on the top level directories from the Sigma community repository rule’s folder.

Next, restart SOCtopus (so-soctopus-restart) and have Playbook pull in the new rules with so-playbook-ruleupdate - this can take a few minutes to complete if pulling in a large amount of new rules.

**Reply by defensivedepth** [reply]
In addition to what @EchoGangster already mentioned, you can try generating an alert for something simple. For instance:

Make sure that the Play that is titled `Security Eventlog Cleared` is `Active`.

Then on a system that is sending windows eventlogs into SO, clear the Security Eventlog. You should see a new event generated in that log with an event id of `1102`. You can also confirm that it is in SO by searching for it in Hunt or Kibana.

The Play should generate an alert for this log within 5 minutes.

**Comment by RoyTM99** [comment]
Hi @EchoGangster / @defensivedepth ,

I will review and test what you tell me and get back to you with news as soon as possible, thank you very much for your support.

Kind Regards
